next.config.mjs

```diff
@@ -20,7 +20,7 @@ const withNextra = nextra({
 
 export default withNextra({
   trailingSlash: false,
-  images: { unoptimized: true },
+  images: { unoptimized: false },
```

Comment:
We can just remove this config?

Reply:
Rather not, as people should be able to easily turn this setting off.